src/app/test-record/notes/notes-edit/notes-edit.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
+import { VIEW_STATE } from '@app/app.enums';
+import { ControlContainer, FormControl, FormGroup, FormGroupDirective } from '@angular/forms';
+import { TestType } from '@app/models/test.type';
+
+@Component({
+  selector: 'vtm-notes-edit',
+  templateUrl: './notes-edit.component.html',
+  styleUrls: ['./notes-edit.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  viewProviders: [{ provide: ControlContainer, useExisting: FormGroupDirective }]
+})
+export class NotesEditComponent implements OnInit {
+  @Input() currentState: VIEW_STATE;
+  @Input() testType: TestType;
+  testResultChildForm: FormGroupDirective;
+  testTypeGroup: FormGroup;
+
+  constructor(parentForm: FormGroupDirective) {
+    this.testResultChildForm = parentForm;
+  }
+
+  ngOnInit() {
+    this.testTypeGroup = this.testResultChildForm.form.get('testType') as FormGroup;
+
+    if (!!this.testTypeGroup) {
```

Comment:
Might need to add a unit test for that we could check that addControl has been called one time